tools/klee-replay/file-creator.c

```diff
@@ -18,7 +18,14 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <termios.h>
-#include <pty.h>
+#ifdef __linux
```

Comment:
This is a hack. Instead once you've figured out which header to include in CMake you should configure `include/klee/Config/config.h.cmin` to define a macro which can be used to decide which header to include.